python_course_1604/class_01/exercize_06_while_break_else.py

```diff
@@ -36,3 +36,10 @@ def count_until(match_value):
 
     Please use while, break, else construct.
     """
+    times = 0
+    while has_next():
+        val = get_next()
+        times += 1
+        if val == match_value:
+            return times
```

Comment:
The behaviour looks correct but I was expecting you would have used break and else constructs. Can you redo this as specified? ;-)